lib/stat_tracker.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/game'
+require './lib/team'
+require './lib/game_team'
+require_relative 'csv_loadable'
+
+class StatTracker
+  include CsvLoadable
+
+  attr_reader :games, :teams, :game_teams
```

Comment:
Are we going to align our attr_reader horizontally or vertically?

Reply:
I prefer vertically, but i would also prefer we agree on the code first and then make it look "pretty" ;)

Reply:
I like vertical

Reply:
I don't mind using either but if people want to use vertical that is cool in my book!